src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -86,6 +86,18 @@ metric. This change only affects users setting `OTEL_SEMCONV_STABILITY_OPT_IN`
 to `http` or `http/dup`.
 ([#4934](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4934))
 
+* **Breaking**: Removed `Enrich` and `Filter` support for **metrics**
```

Comment:
I think its best to explicitly mention `AspNetCoreMetricsInstrumentationOptions` is no longer available as the only reason for its existence (enrich/filter) were removed.